lib/elastomer/client/index.rb

```diff
@@ -533,6 +533,17 @@ def delete_by_query(query, params = nil)
         docs.delete_by_query(query, params)
       end
 
+      # Constructs a Percolator with the given id on this index.
+      #
+      # Examples
+      #
+      #   index.percolator "1"
+      #
+      # Returns a Percolator
+      def percolator(id)
+        Percolator.new(@client, @name, id)
```

Comment:
Since elastomer-client aims to provide a one-to-one mapping to the ElasticSearch API endpoints, I think removing `Client#warmer` is the better option.